mw-webapp/src/utils/ThemeWorker.ts

```diff
@@ -73,7 +73,8 @@ export class ThemeWorker {
    */
   public static loadTheme() {
     const theme = localStorageWorker.getItemByKey("theme");
-    this.setTheme(theme ?? DEFAULT_THEME);
+    // TODO: font use as
```

Comment:
fondt -> dont

Reply:
emm but there wasnt 'as' operator before, why did you add it?

Reply:
deleted